test/message/test_runner_output.js

```diff
@@ -328,3 +328,13 @@ test('subtest sync throw fails', async (t) => {
     throw new Error('thrown from subtest sync throw fails at second');
   });
 });
+
+test('timed out test', { timeout: 500 }, async (t) => {
```

Comment:
I think we can make this value a lot lower (leave the `setTimeout()` delay as is though). The test will run faster and be less likely to have issues in the CI.